### PR TITLE
README, CONTRIBUTING and ISSUE_TEMPLATE, Discord

### LINKED ISSUE #944: Discord Server
Would you like me to set up a Discord server to be able to better communicate with your users?

### PR DESCRIPTION
**Pull request by j2ghz**
Revise the way users are directed when coming to report a problem or request a feature.

Closes #944 

Suggestions welcome, waiting for reviews.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
+| Build | Download | F-Droid | Contact |
+|-------|----------|---------|---------|
+| [![TeamCity (simple build status)](https://img.shields.io/teamcity/https/teamcity.kanade.eu/s/tachiyomi_Build.svg)](https://teamcity.kanade.eu/project.html?projectId=tachiyomi) [![Travis](https://img.shields.io/travis/inorichi/tachiyomi.svg)](https://travis-ci.org/inorichi/tachiyomi) | [![stable release](https://img.shields.io/github/release/inorichi/tachiyomi.svg?maxAge=3600&label=stable)](https://github.com/inorichi/tachiyomi/releases) [![latest dev build](https://img.shields.io/badge/dev-latest%20build-blue.svg)](http://tachiyomi.kanade.eu/latest/app-debug.apk) | [![fdroid release](https://img.shields.io/badge/stable-f--droid.org-blue.svg)](https://f-droid.org/repository/browse/?fdid=eu.kanade.tachiyomi) [![fdroid dev](https://img.shields.io/badge/dev-wiki-blue.svg)](//github.com/inorichi/tachiyomi/wiki/FDroid-for-dev-versions) | [![Discord](https://img.shields.io/discord/349436576037732353.svg)](https://discord.gg/WrBkRk4) |
 
-## [Report an issue](https://github.com/inorichi/tachiyomi/blob/master/.github/CONTRIBUTING.md)
+### **Contact us on [Discord](https://discord.gg/WrBkRk4)**
```
Comment (j2ghz) [inline README.md]: Or 'Discord server available' ?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,24 +1,27 @@
+1. **Before reporting a new issue, take a look at the [FAQ](https://github.com/inorichi/tachiyomi/wiki/FAQ), the [changelog](https://github.com/inorichi/tachiyomi/releases) and the already opened [issues](https://github.com/inorichi/tachiyomi/issues).**
+2. If you are unsure, ask here: [![Discord](https://img.shields.io/discord/349436576037732353.svg)](https://discord.gg/WrBkRk4)
+3. What is your type of issue?
+    * [Catalogue request](#catalogue-requests)
+    * [Bugs](#bugs)
+    * [Feature requests](#feature-requests)
+    * [Translations](https://github.com/inorichi/tachiyomi/wiki/Translation)
+4. After following 1. and 3. you can [open your issue](https://github.com/inorichi/tachiyomi/issues/new)
```
Comment (j2ghz) [inline .github/CONTRIBUTING.md]: Is this understandable enough?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -1,24 +1,27 @@
+1. **Before reporting a new issue, take a look at the [FAQ](https://github.com/inorichi/tachiyomi/wiki/FAQ), the [changelog](https://github.com/inorichi/tachiyomi/releases) and the already opened [issues](https://github.com/inorichi/tachiyomi/issues).**
```
Comment (j2ghz) [inline .github/CONTRIBUTING.md]: Should it also say 'have a look at the dev release, maybe your issue was already fixed'?

File: README.md (hunk 1 above)
Comment (inorichi) [reply README.md]: Contact looks better IMO.

**Pull request merged**